pkg/volume/csi/csi_block.go

```diff
@@ -299,6 +301,13 @@ func (m *csiBlockMapper) SetUpDevice() error {
 	// Call NodeStageVolume
 	_, err = m.stageVolumeForBlock(ctx, csiClient, accessMode, csiSource, attachment)
 	if err != nil {
+		if volumetypes.IsOperationFinishedError(err) {
```

Comment:
Do we need to introduce uncertain device tracking for block volume calls? We can do this in a follow up but we should track that work somewhere?

Reply:
Yes, we should. Filled https://github.com/kubernetes/kubernetes/pull/88016

Reply:
Wait is that correct? That is another PR..

Reply:
sorry, I fixed my previous comment with the right link